design-proposals/klip-62-ksqldb-ruby-client.md

```diff
@@ -0,0 +1,53 @@
+# KLIP 62 - ksqlDB Ruby Client
+
+**Author**: Lapo Elisacci (@LapoElisacci) | 
+**Release Target**: TBD | 
+**Status**: _In Discussion_ | 
+**Discussion**: TBD
+
+**tl;dr:** Develop a Ruby HTTP2 Client to request ksqlDB REST API
+
+## Motivation and background
+
+Allow Ruby / Ruby on Rails developers to get started with ksqlDB.
+
+## What is in scope
+
+* A lightweight but complete and configurable Ruby client.
+* Support for both synchronous and asynchronous queries.
```

Comment:
instead of sync + async queries, we can point out the types of queries (persistent, transient, pull) and if we want to provide sync+async versions of all of them